samples/system-test/detect.v2beta1_test.js

```diff
@@ -0,0 +1,179 @@
+    );
+  }
+);
+
+test.serial(
+  `It should locate knowledge base with listKnowledgeBases`,
+  async t => {
+    const output = await runAsync(`${cmd} listKnowledgeBases`);
+    const parsedOut = output.split(`\n`);
+    parsedOut.forEach(p => {
+      if (p.includes(testKnowledgeBaseName)) t.pass();
+    });
+    console.log(output);
+  }
+);
+
+test.serial(`It should create a document in Knowledge Base`, async t => {
+  const output = await runAsync(
+    `${cmd} createDocument -n "${knowbaseFullName}" -z "${testDocumentPath}" -m "${testDocName}"`
+  );
+  t.true(output.includes(`Document created`));
+  console.log(`Document created successfully`);
+});
+
+test.serial(
+  `It should list document just created with listDocuments`,
+  async t => {
+    const output = await runAsync(
+      `${cmd} listDocuments -n "${knowbaseFullName}"`
+    );
+    const parsedOut = output.split(`\n`);
+    documentFullPath = parsedOut[parsedOut.length - 1].split(`:`)[1];
+    t.true(documentFullPath !== ``);
+    console.log(`Confirming documentID: ${documentFullPath}`);
+  }
+);
+
+test.serial(`It should detect Intent with Text to Speech Response`, async t => {
+  const output = await runAsync(
+    `${cmd} detectIntentwithTexttoSpeechResponse -q "${testQuery}"`
+  );
+  const parsedOut = output.split(`\n`);
+  t.true(
+    parsedOut[1].includes(
+      `Audio content written to file: ./resources/output.wav`
+    )
+  );
+  console.log(`confirm audio file generated: ${parsedOut[1]}`);
+});
+
+test.serial(`It should detect Intent Knowledge`, async t => {
+  const output = await runAsync(
+    `${cmd} detectIntentKnowledge -q "${testQuery}"`
+  );
+  const parsedOut = output.split(`\n`);
+  // documentFullPath = parsedOut[parsedOut.length - 1].split(`:`)[1];
+  t.true(parsedOut[parsedOut.length - 3] !== null);
+  console.log(`confirm query result: ${parsedOut[parsedOut.length - 5]}`);
+});
+
+test.serial(`It should detect sentiment with intent`, async t => {
+  const output = await runAsync(
+    `${cmd} detectIntentandSentiment -q "${testQuery}"`
+  );
+  const parsedOut = output.split(`\n`);
+  t.true(parsedOut[parsedOut.length - 3].includes(`Detected sentiment`));
+  console.log(`confirm sentiment result: ${parsedOut[parsedOut.length - 2]}`);
+  console.log(`confirm sentiment result: ${parsedOut[parsedOut.length - 1]}`);
+});
+
+test.serial(`It should detect Intent with Model Selection`, async t => {
+  const output = await runAsync(`${cmd} detectIntentwithModelSelection`);
+  const parsedOut = output.split(`\n`);
+  t.true(parsedOut[1] !== null);
+  console.log(`ParsedOut[1] is: ${parsedOut[1]}`);
+});
+
+test.serial(
+  `It should delete document just created with deleteDocument`,
+  async t => {
+    console.log(`Deleting document: ${documentFullPath}`);
+    const output = await runAsync(
+      `${cmd} deleteDocument -d "${documentFullPath}"`
+    );
+    t.true(output.includes(`document deleted`));
+  }
+);
+
+// test.serial(
```

Comment:
Why is there commented out code in the diff?

Remove all extraneous commented code.